logic/src/commonMain/kotlin/com/wire/kalium/logic/feature/conversation/PersistMigratedConversationUseCase.kt

```diff
@@ -24,15 +27,18 @@ fun interface PersistMigratedConversationUseCase {
 }
 
 internal class PersistMigratedConversationUseCaseImpl(
-    private val conversationRepository: ConversationRepository
+    private val migrationDAO: MigrationDAO,
```

Comment:
Do we need a repository in between here ? 🤔  I don't see much benefit of it, but if you do go ahead :D

Reply:
I don't think, so it is a simple insert or ignore operation that I wanted to isolate from the rest of the app. i am considering doing the same for inserting users as well but that will be in a later PR